styles.css

```diff
@@ -0,0 +1,71 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap");
+
+* {
+  margin: 0;
+  font-family: Poppins, sans-serif;
+}
+
+body {
+  background-color: #f4f4f4;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 36px 24px;
+  background-color: #fff;
+}
+
+header .nav-brand {
+  color: #6070ff;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  text-align: center;
+}
+
+section.headerline {
+  background-image: url("./images/icons/header-shapes.svg");
+  background-repeat: no-repeat;
+  height: 68vh;
+  width: 89%;
+  padding: 14vh 24px 16px;
```

Comment:
- [ ] Great job using flexible units but there seems to be an unnecessary space on your site, Kindly style the section so that it looks similar to the design.

Your Implementation | Figma Desgin
:---: | :---:
![image](https://user-images.githubusercontent.com/59416136/153271246-78dfcdf1-5c77-4a1f-9814-a398637035c1.png) | ![image](https://user-images.githubusercontent.com/59416136/153271361-ab92cacf-5eb4-4302-93bc-48d17aa191c2.png)

Reply:
👍🏽 UI looks good now.